src/Raven.Server/ServerWide/Commands/DeleteDatabaseCommand.cs

```diff
@@ -14,12 +14,11 @@ public class DeleteDatabaseCommand : UpdateDatabaseCommand
         public string[] FromNodes;
         public bool UpdateReplicationFactor = true;
 
-        public DeleteDatabaseCommand() : base(null)
+        public DeleteDatabaseCommand()
```

Comment:
ErrorOnDatabaseDoesNotExists will be `false` now?

Reply:
This `ctor` is only for de-serialization. No code path uses it. 
What we do missing is to include the `ErrorOnDatabaseDoesNotExists ` in the `ToJson` of the `UpdateDatabaseCommand`

Reply:
Depends for what purpose it is used. You know better.